03_Modules/SmartCharging/src/module/api.ts

```diff
@@ -78,12 +97,34 @@ export class SmartChargingModuleApi
     CallAction.GetChargingProfiles,
     GetChargingProfilesRequestSchema,
   )
-  getChargingProfile(
+  async getChargingProfile(
     identifier: string,
     tenantId: string,
     request: GetChargingProfilesRequest,
     callbackUrl?: string,
   ): Promise<IMessageConfirmation> {
+    if (
```

Comment:
If ChargingProfileId is not set. Don't we have to validate that one of the other conditions is met according to `K09.FR.03` :

> The CSMS SHALL specify in chargingProfile criteria in GetChargingProfilesRequest either:
> - a (list of) chargingProfileId(s) OR
> - one or more of the fields stackLevel, chargingLimitSource, chargingProfilePurpose.